drivers/led_strip/Kconfig.ws2812

```diff
@@ -7,50 +7,48 @@
 #
 # https://wp.josh.com/2014/05/13/ws2812-neopixels-are-not-so-finicky-once-you-get-to-know-them/
 
-menuconfig WS2812_STRIP
-	bool "WS2812 (and compatible) LED strip driver"
-	help
-	  Enable LED strip driver for daisy chains of WS2812-ish (or WS2812B,
-	  WS2813, SK6812, Everlight B1414, or compatible) devices.
-
-choice WS2812_STRIP_DRIVER
-	prompt "Driver backend"
-	default WS2812_STRIP_SPI
-	depends on WS2812_STRIP
-
 config WS2812_STRIP_SPI
-	bool "SPI driver"
+	bool "WS2812 LED strip SPI driver"
+	default y
+	depends on DT_HAS_WORLDSEMI_WS2812_SPI_ENABLED
 	depends on SPI
```

Comment:
We're now allowed to use `select SPI`, this will allow some additional clean up.

Reply:
I got it. Add `select SPI` and remove duplicate from *.conf.